timetracker/account/views.py

```diff
@@ -1,10 +1,11 @@
 import logging
-
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.auth.views import LoginView as BaseLoginView
 from django.views.generic import TemplateView
-
-
+from django.contrib.auth import login, authenticate
+from django.shortcuts import redirect
+from django.views import generic
+from account import forms
 logger = logging.getLogger(__name__)
```

Comment:
Add 2 lines between imports and the rest of the code.
```suggestion


logger = logging.getLogger(__name__)
```